content/docs/reference/api-docs.md

```diff
@@ -190,9 +187,7 @@ description: >-
             <td>
               <code>issuerRef</code>
               <br />
-              <em>
-                <a href="#meta.cert-manager.io/v1.ObjectReference">ObjectReference</a>
-              </em>
+              <em>github.com/cert-manager/cert-manager/pkg/apis/meta/v1.IssuerReference</em>
```

Comment:
Not sure why the generator no longer generates html links using anchors. It now shows like this:

<img width="1262" height="314" alt="Image" src="https://github.com/user-attachments/assets/e5c166e3-9ea8-4ff3-9dbb-16219dd7a3e7" />

Not a massive deal. Would it be easy to fix that right away, or does it require further investigation?